src/main/kotlin/calculator/CustomStringDelimiter.kt

```diff
@@ -0,0 +1,50 @@
+package calculator
+
+import kotlin.streams.toList
+
+class CustomStringDelimiter(value: String?) {
+
+    private val values: List<String>
+    private val operands: MutableList<Int>
+    private val operators: MutableList<String>
+
+    init {
+
+        values = validate(value)
+        operators = values.stream()
+            .filter { Expression.STRING_OPERATORS.contains(it) }
+            .toList()
+            .toMutableList()
```

Comment:
```suggestion
        operators = values.filter { Expression.STRING_OPERATORS.contains(it) }.toMutableList()
```

자바로 평소에 쓰시다 보니 stream을 쓰셨군요! 코틀린은 Collections와 Sequences가 있는데요.
흔히 자바에서 사용한 stream의 lazy evaluation는 코틀린에서 Sequences API를 통하여 가능합니다.
그리고, Eegar evaluation는 Collections API만으로도 충분합니다.

[해당 글](https://codechacha.com/ko/kotlin-sequences/)을 읽어보시고, 코틀린에서는 Java의 Stream 대신 무엇을 사용하는지 확인해 보셨으면 합니다.